### ISSUE TITLE
Splitter value should be updated every epoch

### ISSUE THREAD

**Issue by Seondong** [Seondong/Customs-Fraud-Detection#40]
Hi, if we want to use the most recent data as a validation set, I think the values in splitter be changed when new weekly data is added to the training set? (active_DATE.py, line 170)
